.doc_gen/metadata/dynamodb_metadata.yaml

```diff
@@ -23,8 +23,26 @@ dynamodb_ExecuteStatement:
             - description:
               snippet_tags:
                 - dynamodb.java2.scenario.partiql.main
+    JavaScript:
+      versions:
+        - sdk_version: 3
+          github: javascriptv3/example_code/dynamodb
+          sdkguide:
+          excerpts:
+            - description: Create the client.
+              snippet_tags:
+                - dynamodb.JavaScript.scenario.partiQL.basics.createclientv3
+            - description: Create the document client.
+              snippet_tags:
+                - dynamodb.JavaScript.scenario.partiQL.createdocclientv3
+            - description: Query single items.
+              snippet_tags:
+                - javascript.dynamodb_scenarios.partiQL_basics
+            - description: Query items by batch.
+              snippet_tags:
+                - javascript.dynamodb_scenarios.partiQL_batch_basics
   services:
-    dynamodb: {ExecuteStatement}
+    dynamodb: {ExecuteStatement, BatchExecuteStatement}

```

Comment:
Can you separate the batch version into a second scenario? That was the original spec that DDB asked for.

Reply:
done